Cargo.toml

```diff
@@ -118,7 +118,7 @@ axum = "=0.7.4" # TODO: 0.7.5+ does not work with current toolchain
 etcd-client = { package = "madsim-etcd-client", version = "0.4" }
 futures-async-stream = "0.2.9"
 hytra = "0.1"
-rdkafka = { package = "madsim-rdkafka", version = "0.3.4", features = [
+rdkafka = { package = "madsim-rdkafka", version = "0.4.0", features = [
```

Comment:
This is to refactor rdkafka.  https://github.com/madsim-rs/madsim/pull/207#event-12723713372

Reply:
bumped to v0.4.1 to add some missing APIs https://github.com/madsim-rs/madsim/pull/208